src/core/AutoRest.Extensions/ParameterGroupExtensionHelper.cs

```diff
@@ -110,9 +110,7 @@ private static string GenerateParameterGroupModelText(IEnumerable<Method> method
             List<Method> methodList = methodsWhichUseGroup.ToList();
             if (methodList.Count == 1)
             {
-                Method method = methodList.Single();
-                return string.Format(CultureInfo.InvariantCulture, "Additional parameters for the {0} operation.",
-                    createOperationDisplayString(method.MethodGroup.Name.ToPascalCase(), method.Name));
+                return "Additional parameters for "+ methodList.Single().Name + " operation.";
```

Comment:
still wondering, why is this inconsistent with the `Documentation` string you assemble above?